client/src/index.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './App';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
```

Comment:
```suggestion
       <App />
 ```
Just add the app in the strict mode tag without router we add it in the app itself